node/package.json

```diff
@@ -32,8 +32,7 @@
     "slugify": "^1.2.6",
     "typescript": "3.8.3",
     "unescape": "^1.0.1",
-    "vtex.search-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.search-graphql@0.32.0/public"
-  },
+    "vtex.search-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.search-graphql@0.33.0-beta.4/public"  },
```

Comment:
Reminder to change this once https://github.com/vtex-apps/search-graphql/pull/91 is merged.